### PR TITLE
integrated first part of SCCS data

### PR DESCRIPTION
**Pull request by xrotwang**
addresses #31

### CODE REVIEW

File: datasets/SCCS/convert.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+from pydplace.api import Society
+
+#
+# read wnai.bib
+# write mapping of bibkeys to bib.json
+# read sources.bib
+# merge records from wnai.bib not yet in sources.bib
+# write sources.bib
+#
```
Comment (SimonGreenhill) [inline datasets/SCCS/convert.py]: Dead comments? or TODOs?

File: datasets/SCCS/convert.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    with open(fname, 'rb') as fp:
+        c = fp.read()
+
+    with open(fname, 'wb') as fp:
+        fp.write(c.replace(b'\x9d', b''))
+
+    for r in reader(fname, dicts=True, encoding='cp1252'):
+        if (r.get('dataset') == 'SCCS') or (r.get('Dataset') == 'SCCS') or (r.get('Datset') == 'SCCS'):
+            yield r
+
```
Comment (SimonGreenhill) [inline datasets/SCCS/convert.py]: Eww cp1252. Could we just convert the file once to utf8 and store that in the repository? Much nicer to have utf8 everywhere?

Comment (SimonGreenhill) [reply datasets/SCCS/convert.py]: ...and that's what the rest of the code is doing, Simon. Nevermind.

File: datasets/SCCS/convert.py (hunk 1 above)
Comment (xrotwang) [reply datasets/SCCS/convert.py]: The whole file is only temporary, until I get the final set of data from @kirbykat

File: datasets/SCCS/convert.py (hunk 2 above)
Comment (xrotwang) [reply datasets/SCCS/convert.py]: Yes, what's stored in the repos is the output of this script.

File: datasets/SCCS/convert.py (hunk 1 above)
Comment (SimonGreenhill) [reply datasets/SCCS/convert.py]: 👍

**Pull request merged**